brave/brave-core
# Add metric for sync join

Resolves https://github.com/brave/brave-browser/issues/41336



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start with fresh profile, ensure metric does not exist in local state.
1. Create a new sync chain. Ensure metric value is 1.
2. Leave the sync chain. Join an existing sync code by inputting a sync code. Ensure metric value is 2.
3. Start again with fresh profile. Execute step 2, ensure metric value is 2.
